libs/wingsdk/src/shared/convert.ts

```diff
@@ -14,18 +13,17 @@ export function convertBetweenHandlers(
   newHandlerClientClassName: string,
   args: Record<string, unknown> = {}
 ): IInflight {
-  const handlerClient = baseHandler._toInflight();
-  const newCode = `\
-new (require("${normalPath(
-    newHandlerClientPath
-  )}")).${newHandlerClientClassName}({ handler: ${handlerClient}, args: ${JSON.stringify(
-    args
-  )} })`;
-
   const newHandler = {
     ...baseHandler,
-    _hash: createHash("md5").update(newCode).digest("hex"),
+    _id: baseHandler._id,
```

Comment:
Shouldn't this generate a new id? Technically this is a new inflight

Reply:
Fixed